deployability/modules/generic/logger/logger.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import logging.config
```

Comment:
You have "import logging" and "import logging.config". Be careful with this. Please choose one.